packages/aws-appsync/src/store.ts

```diff
@@ -7,7 +7,7 @@ import { applyMiddleware, createStore, compose, combineReducers, Store, Reducer,
 import { offline } from '@redux-offline/redux-offline';
 import defaultOfflineConfig from '@redux-offline/redux-offline/lib/defaults';
 import { PERSIST_REHYDRATE } from "@redux-offline/redux-offline/lib/constants";
-import { KEY_PREFIX as REDUX_PERSIST_KEY_PREFIX } from "redux-persist/constants";
+import { KEY_PREFIX as REDUX_PERSIST_KEY_PREFIX } from "redux-persist/lib/constants";
```

Comment:
Can you try:
`import { KEY_PREFIX as REDUX_PERSIST_KEY_PREFIX } from "redux-persist";`

I believe the constants are exported from the index of redux-persist.

Reply:
Actually, I believe this was added in v5, we are using v4.

Reply:
Hi,

I tried to apply you suggestion and I get the following TS error:

`src/store.ts:10:10 - error TS2305: Module '"redux-persist"' has no exported member 'KEY_PREFIX'.`

So that does not seem to be possible...

I also did more tests by modifying the the **aws-appsync** package in my App:
- The change with **var constants_2 = require("redux-persist/lib/constants")** is working both with redux-pesist v5 and v4
- If I change  with **var constants_2 = require("redux-persist")**, it is working as well with redux-pesist v5 and v4
- The generated code **var constants_2 = require("redux-persist/constants")** from the current version of **aws-appsync** is actually working but only with redux-persist v4 and I think I had installed or had another dependency that forced me to have version 5.

So, finally, I think the PR should stay like that. Do you agree?

Reply:
Let's marge this fix finally !

Reply:
When ll u merge this.? still getting this issue.